packages/date-time/src/components/WeeklyDayPicker/WeeklyDayPicker.base.tsx

```diff
@@ -242,12 +258,20 @@ export class WeeklyDayPickerBase extends BaseComponent<IWeeklyDayPickerProps, IW
     );
   };
 
-  private _onSelectPrevWeek = () => {
-    this._navigateDate(addDays(this.state.navigatedDate, -7));
+  private _onSelectPrev = () => {
```

Comment:
nit: could we maintain the specificity of the function name here, something like `_onSelectPrevDateRange`? same applies to `_onSelectNext'